core/frontend/src/RealityDataSource.ts

```diff
@@ -50,13 +50,13 @@ export interface RealityDataSource {
    */
   getTileJson(name: string): Promise<any>;
 }
-/** @alpha */
+/** Utility functions for RealityDataSource
+ * @beta
+ */
 export namespace RealityDataSource {
-  /** Utility function to convert a RealityDataSourceKey into its string representation
-  * @alpha
-  */
+  /** Utility function to convert a RealityDataSourceKey into its string representation */
   export function keyToString(rdSourceKey: RealityDataSourceKey): string {
-    return `${rdSourceKey.provider}:${rdSourceKey.format}:${rdSourceKey.id}:${rdSourceKey.iTwinId}`;
+    return rdSourceKey.toString();
```

Comment:
What is the purpose of having this separate function that just calls `toString` for you?

Reply:
removed